message/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class AuthorOptions(models.TextChoices):
+    DogOwner = 'O', 'DogOwner'
+    DayCare = 'D', 'DayCare'
+
+
+class Message(models.Model):
+    author = models.CharField(max_length=1,
+                              choices=AuthorOptions.choices)
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    text = models.TextField()
+    date = models.DateTimeField(default=timezone.now)
+    message_reply = models.ForeignKey('self', on_delete=models.CASCADE, null=True)
```

Comment:
what is this for?

Reply:
all the fields are according the ERD

Reply:
I will be nice if you could give short explanation :slightly_smiling_face:

Reply:
@Yuval-Vino that doesn't mean it is needed.
if the message class exists for a chat - there is no reason to include a specific reply.